src/models/accounts.nim

```diff
@@ -2,6 +2,7 @@ import eventemitter
 import json_serialization
 import ../status/accounts as status_accounts
 import ../status/types
+import ../status/libstatus
```

Comment:
importing libstatus outside of `status/` is forbidden :)

Reply:
Yeah I know, but since it was only until we fix the TODO it could be accepted 🤷